model/event/subscriptions.go

```diff
@@ -530,6 +530,7 @@ func RemoveSubscription(id string) error {
 	})
 }
 
+// IsSubscriptionAllowed disallows subscriptions that abuse Jira or webhooks.
```

Comment:
Can we absorb this into Validate rather than have its own exported function?

Reply:
Yeah seems reasonable; I want to avoid doing too much cleanup work here since subscriptions are their own world, but this function isn't individually unit tested or anything so idk why it lives alone 🤷‍♀️

Reply:
I also removed some of the old logic; I think it's trying to prevent us from commenting on Jira or creating a webhook for particular versions,tasks,builds, but it doesn't seem to work, and there's no reason for that really; this logic is a bit stronger.

Reply:
Jk I'm going to keep it in;  I think it's actually checking something different (there are certain options, like creating a jira issue, that we don't show on the new UI even for regular tasks)

Reply:
Makes sense, good catch!